src/common/VFileIO.cpp

```diff
@@ -1296,15 +1296,26 @@ void XMFS::init(const std::string &AppDir,
       break;
     }
   }
+  /* Try some default fallbacks */
   if (!m_bGotSystemDataDir) {
-    m_SystemDataDir = std::string("/usr/share/");
-    if (isDir(m_SystemDataDir)) {
-      m_bGotSystemDataDir = true;
+    const std::vector<std::string> dataDirs = {
```

Comment:
macOS and FreeBSD prefer /usr/local over /usr, this allows xmoto to properly reflect that without hackish workarounds